geps/overview.md

```diff
@@ -151,6 +151,7 @@ use the `experimental` Golang build tag to denote experimental functionality.
 Some other requirements must be met before marking a GEP `Experimental`:
 
 - the graduation criteria to reach `Standard` MUST be filled out
+- the GEP must have at least one Feature Name for features described inside.
```

Comment:
Maybe add some clarification ~somewhere that this only applies to GEPs that will have corresponding conformance tests. Although rare, there are some edge cases like policy attachment that won't have a feature name or conformance tests.